src/web/event-notifications/event-notification-types/TeamsNotificationForm.tsx

```diff
@@ -0,0 +1,189 @@
+                      + 'Message:              ${event.message}\n'
+                      + 'Source:               ${event.source}\n'
+                      + 'Key:                  ${event.key}\n'
+                      + 'Priority:             ${event.priority}\n'
+                      + 'Alert:                ${event.alert}\n'
+                      + 'Timestamp Processing: ${event.timestamp}\n'
+                      + 'Timerange Start:      ${event.timerange_start}\n'
+                      + 'Timerange End:        ${event.timerange_end}\n'
+                      + 'Event Fields:\n'
+                      + '${foreach event.fields field}\n'
+                      + '${field.key}: ${field.value}\n'
+                      + '${end}\n'
+                      + '${if backlog}\n'
+                      + '--- [Backlog] ----------\n'
+                      + 'Last messages accounting for this alert:\n'
+                      + '${foreach backlog message}\n'
+                      + '${message.timestamp}  ::  ${message.source}  ::  ${message.message}\n'
+                      + '${message.message}\n'
+                      + '${end}'
+                      + '${end}\n',
+      /* eslint-enable no-template-curly-in-string */
+      icon_url: '',
+      backlog_size: 0,
+
+    };
+    constructor(props: TeamsNotificationFormType | Readonly<TeamsNotificationFormType>) {
+      super(props);
+
+      const defaultBacklogSize = props.config.backlog_size;
+
+      this.state = {
+        isBacklogSizeEnabled: defaultBacklogSize > 0,
+        backlogSize: defaultBacklogSize,
+      };
+    }
+
+    handleBacklogSizeChange = (event: { target: { name: string; }; }) => {
+      const { name } = event.target;
+      const value = getValueFromInput(event.target);
+
+      this.setState({ [camelCase(name)]: value });
+      this.propagateChange(name, getValueFromInput(event.target));
+    };
+
+    toggleBacklogSize = () => {
+      const { isBacklogSizeEnabled, backlogSize } = this.state;
+
+      this.setState({ isBacklogSizeEnabled: !isBacklogSizeEnabled });
+      this.propagateChange('backlog_size', (isBacklogSizeEnabled ? 0 : backlogSize));
+    };
+
+    propagateChange = (key: string, value: any) => {
+      const { config, onChange } = this.props;
+      const nextConfig = cloneDeep(config);
+      nextConfig[key] = value;
+      onChange(nextConfig);
+    };
+
+    handleColorChange:  (color: string, _: any, hidePopover: any) => void= (color, _, hidePopover) => {
+      hidePopover();
+      this.propagateChange('color', color);
+    };
+
+    handleChange = (event: { target: { name: any; }; }) => {
+      const { name } = event.target;
+      this.propagateChange(name, getValueFromInput(event.target));
+    };
+
+    render() {
+      const { config, validation } = this.props;
+      const { isBacklogSizeEnabled, backlogSize } = this.state;
+      const url = 'https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications';
```

Comment:
With the documentation system change, the closest match seems to be https://docs.graylog.org/docs/alerts#notifications .